src/realm/sync/protocol.hpp

```diff
@@ -63,14 +63,16 @@ namespace sync {
 //   14 Support for server initiated bootstraps, including bootstraps for role/
 //      permissions changes instead of performing a client reset when changed.
 //
+//   15 Support for relaxed (flexible) schemas
+//
```

Comment:
I don't have a problem with doing this early, but worth noting that the server does not yet know about this protocol version (and shouldn't need to for this piece of relaxed schema)

Reply:
Should we delay the protocol bump until later? I believe the server will just negotiate to v14 and there is nothing in the sync client or relaxed schema implementation that cares whether it's 14 or 15.

(Also this is merging to a feature branch, so it will be a while before it hits `master`)

Reply:
I think generally we wait until the project is more or less complete incase some other feature needs to take the earlier protocol version, but I don't feel strongly about waiting since this is a feature branch. Worst case you'll need to increment it by 1 at a later point

Reply:
Reverted protocol change